src/resolve/resolved-dns-server.h

```diff
@@ -100,6 +100,9 @@ struct DnsServer {
         /* If linked is set, then this server appears in the servers linked list */
         bool linked:1;
         LIST_FIELDS(DnsServer, servers);
+
+        /* Set when the server is configured manually via D-Bus, to avoid removing it on reload */
+        bool from_dbus:1;
```

Comment:
Please reorder elements with bitfield specifier `:1` to pack them more memory efficient.

Reply:
Or, maybe, better to make it enum, to explain the source of the server:
```
DNS_SERVER_FROM_CONFIG_FILE,
DNS_SERVER_FROM_NETWORKD,
DNS_SERVER_FROM_DBUS,
```